src/test/java/org/openlmis/fulfillment/web/OrderControllerTest.java

```diff
@@ -186,6 +187,8 @@ public void shouldDeleteMultipleOrders() {
     ids.add(order.getId());
     ids.add(orderTwo.getId());
 
+    IdsDto idsDto = new IdsDto(ids);
```

Comment:
Fix checkstyle error:
[ant:checkstyle] [ERROR] /app/src/test/java/org/openlmis/fulfillment/web/OrderControllerTest.java:190: Distance between variable 'idsDto' declaration and its first usage is 6, but allowed 3.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value). [VariableDeclarationUsageDistance]